distributions/aws/examples/storage/fsx-for-lustre/README.md

```diff
@@ -0,0 +1,183 @@
+```
+aws fsx describe-file-systems --query "FileSystems[*].FileSystemId" --output text --region $CLUSTER_REGION
+```
+
+```
+export file_system_id=<fsx-id-to-use>
+```
+
+2. Once you have the filesystem id, Use the following command to retriev DNSName, and MountName values.
+```
+export dns_name=$(aws fsx describe-file-systems --file-system-ids $file_system_id --query "FileSystems[0].DNSName" --output text --region $CLUSTER_REGION)
+
+export mount_name=$(aws fsx describe-file-systems --file-system-ids $file_system_id --query "FileSystems[0].LustreConfiguration.MountName" --output text --region $CLUSTER_REGION)
+```
+
+3. Now edit the sample/fsx-pv.yaml to replace <file_system_id>, <dns_name>, and <mount_name> with your values.
+```
+yq e '.spec.csi.volumeHandle = env(file_system_id)' -i fsx-for-lustre/static-provisioning/fsx-pv.yaml
+yq e '.spec.csi.volumeAttributes.dnsname = env(dns_name)' -i fsx-for-lustre/static-provisioning/fsx-pv.yaml
+yq e '.spec.csi.volumeAttributes.mountname = env(mount_name)' -i fsx-for-lustre/static-provisioning/fsx-pv.yaml
+```
+
+4. The `PersistentVolume` is a cluster scoped resource but the `PersistentVolumeClaim` needs to be in the namespace you will be accessing it from. Be sure to replace the `kubeflow-user-example-com` namespace specified in the `fsx-for-lustre/static-provisioning/fsx-pvc.yaml` file with the namespace for the kubeflow user. 
+
+5. Now create the required persistentvolume and persistentvolumeclaim resources as -
+```
+kubectl apply -f fsx-for-lustre/static-provisioning/fsx-pv.yaml
+kubectl apply -f fsx-for-lustre/static-provisioning/fsx-pvc.yaml
+```
+
+## 5.2 Check your Setup
+Use the following commands to ensure all resources have been deployed as expected and the PersistentVolume is correctly bound to the PersistentVolumeClaim
+```
+kubectl get pv
+
+NAME    CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                 STORAGECLASS   REASON   AGE
+fsx-pv  1200Gi     RWX            Recycle          Bound    kubeflow-user-example-com/fsx-claim                           11s
+```
+
+```
+kubectl get pvc -n kubeflow-user-example-com
+
+NAME        STATUS   VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+fsx-claim   Bound    fsx-pv   1200Gi     RWX                           83s
+```
+
+Now, Port Forward as needed and Login to the Kubeflow dashboard. You can also check the `Volumes` tab in Kubeflow and you should be able to see your PVC is available for use within Kubeflow. 
+In the following two sections we will be using this PVC to create a notebook server with Amazon FSx mounted as the workspace volume, download training data into this filesystem and then deploy a TFJob to train a model using this data. 
+
+## 5.3 Using FSx volume as workspace or data volume for a notebook server 
```

Comment:
looks like duplication starting from section 5.1

can these instructions atleast from 5.3 live under a common readme under storage? since you just need the CLAIM_NAME from this point onwards

Reply:
I considered this, but ideally, the fsx sample should not be the same as the efs one. The main advantage of fsx over efs  is the ability to use data directly from s3 and I want to change this sample to do that instead and thus decided to keep them seperate for now. In either case the files in the training-sample will remain the same.